src/core/ui/components/sign-in/sign-in-buttons.tsx

```diff
@@ -25,65 +16,12 @@ type Props = {
 };
 
 export const SignInButtons = ({ showAllWallets, onClick }: Props) => {
-  const { excludeLoginMethodTypes, includeLoginMethodTypes, viewOnly } =
-    useSignInContext();
-  const user = useUser();
-  const enabledLoginMethods = useLoginMethods();
+  const { web2, web3 } = useLoginMethods();
```

Comment:
Moving all this logic to the hook to reuse it and have cleaner components

Reply:
Will that work? I have a feeling there was a reason it was split out like this... I dont' recall for sure though so if this works then sounds good.

Reply:
Yeah, tested both the new flow and the old one and it works in both.